thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/AlertFilterType.java

```diff
@@ -1,5 +1,5 @@
 package com.linkedin.thirdeye.detector.email.filter;
 
 public enum AlertFilterType {
-  DUMMY, ALPHA_BETA
+  DUMMY, ALPHA_BETA, ALPHA_BETA_LOGISTIC
```

Comment:
If every alert filter is accessed through alert filter factory, which uses java reflection to read the information of filters, then this class could be removed